### PR TITLE
{vis}[GCCcore/8.2.0] GtkSourceView v3.24.11, GtkSourceView v4.4.0

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GtkSourceView/GtkSourceView-3.24.11-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ('GObject-Introspection', '1.60.1', '-Python-3.7.2'),
+]
+
+dependencies = [
+    ('GLib', '2.60.1'),
+    ('GTK+', '3.24.8'),
+    ('libxml2', '2.9.8'),
+]
+
+configopts = "--disable-silent-rules --enable-introspection=yes "
```
Comment (smoors) [inline easybuild/easyconfigs/g/GtkSourceView/GtkSourceView-3.24.11-GCCcore-8.2.0.eb]: @Flamefire just a question: why don't you include these options in version 4.4.0 as well?

Comment (Flamefire) [reply easybuild/easyconfigs/g/GtkSourceView/GtkSourceView-3.24.11-GCCcore-8.2.0.eb]: Because this uses ConfigureMake, the other Meson

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/379521fe3a202591c783531176230fb5 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Flamefire!

**Pull request merged**